third_party/move/move-vm/types/src/instr.rs

```diff
@@ -124,6 +124,7 @@ pub enum Instruction {
     CastI128,
     CastI256,
     Negate,
+    AbortMsg,
```

Comment:
One ugliness with this indirection is that the spec is gone (which is in file format)...   
  
I think you should have placeholder at least until the real Bytecode is added, saying whats expected on the stack in which order.